### PR TITLE
Towards 100% coverage

### PR DESCRIPTION
**Pull request by RemDelaporteMathurin**
## Proposed changes

In the test suite, plenty of error raises weren't tested. This aims to increase the code coverage in order to fully test the code.
By increasing the coverage, bugs that weren't caught by the CI have now arised:
- #420 
- #418 
- #417 
- #413 
- #411 

To 100% and beyond ⚡ 

## Types of changes

What types of changes does your code introduce to the Paramak?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Code refactoring
- [ ] Documentation Update (if none of the other choices apply)
- [x] New tests

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._

- [x] Pep8 applied
- [x] Unit tests pass locally with my changes
- [x] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

## Further comments

Known issues:
- there are plenty of tests in test_parametric_reactors/ that randomely fail. I wonder if it's related to #409

### CODE REVIEW

File: tests/test_parametric_reactors/test_SubmersionTokamak.py [hunk 1]
```diff
@@ -410,11 +386,36 @@ def warning_trigger():
...
-            assert len(w) == 1
-            assert issubclass(w[-1].category, UserWarning)
-            assert "360 degree rotation may result in a Standard_ConstructionError or AttributeError" in str(
-                w[-1].message)
+
+    def test_SubmersionTokamak_error_divertor_pos(self):
+        test_reactor = self.SingleNullSubmersionTokamak
+
+        def invalid_divertor_position():
+            test_reactor.divertor_position = "coucou"
```
Comment (Shimwell) [inline tests/test_parametric_reactors/test_SubmersionTokamak.py:398]: coucou

File: tests/test_parametric_reactors/test_SubmersionTokamak.py [hunk 2]
```diff
@@ -410,11 +386,36 @@ def warning_trigger():
...
+
+        self.assertRaises(ValueError, invalid_divertor_position)
+
+        def invalid_support_position():
+            test_reactor.support_position = "coucou"
```
Comment (Shimwell) [inline tests/test_parametric_reactors/test_SubmersionTokamak.py:403]: coucou

**Review by Shimwell — APPROVED** [review]
Looks great to me, sooooo many tests

**Comment by RemDelaporteMathurin** [comment]
> Looks great to me, sooooo many tests

Glad you're happy about it.  Without the neutronics, the coverage is 99.6% !!!!

There were a few things I couldn't test like the moab import error.... Similarly, I didn't want to mess with @billingsley-john 's cutting wedge function in utils😄 

Shall I merge ?

**Pull request merged**